src/coreclr/ilasm/asmparse.y

```diff
@@ -47,8 +47,8 @@
 %token <string>  DOTTEDNAME     /* System.Object */
 %token <binstr>  QSTRING        /* "Hello World\n" */
 %token <string>  SQSTRING       /* 'Hello World\n' */
-%token <int32>   INT32          /* 3425 0x34FA  0352  */
-%token <int64>   INT64          /* 342534523534534      0x34FA434644554 */
+%token <int32>   INT32_V        /* 3425 0x34FA  0352  */
+%token <int64>   INT64_V        /* 342534523534534      0x34FA434644554 */
```

Comment:
We don't have to change it if we use byacc instead of bison.
And worth to note by using bison there will be license conflict between GPL and MIT. So I would recommend to revert this change and use byacc instead.

Reply:
[BISON Conditions](https://www.gnu.org/software/bison/manual/html_node/Conditions.html)

I'm not a lawyer. @richlander , what do you think? This only applies to the parser the BISON generates from the grammar.

Reply:
Reverting this can help prevent developers from generating code using bison by accident (because it would just fail), which can make sure we won't have license issues.

Reply:
I think it would be better to use byacc due to the licensing personally.